Stratis.Bitcoin/Builder/FullNodeBuilder.cs

```diff
@@ -49,8 +49,9 @@ public FullNodeBuilder(NodeSettings nodeSettings)
 			List<Action<IFeatureCollection>> featuresRegistrationDelegates, IFeatureCollection features)
 			: this(configureServicesDelegates, configureDelegates, featuresRegistrationDelegates, features)
 		{
-			this.NodeSettings = nodeSettings ?? NodeSettings.Default();
-			this.Network = nodeSettings.GetNetwork();
```

Comment:
Since this is in a ctor, I guess I can, since this.NodesSettings has a public setter I often would not want to do that since it could be null before we set it......